### ISSUE TITLE
Volume setting for signals?

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#213619]
Testing #213344

Why are we in the business of configuring volume levels?
IMHO this should be an OS thing - user can configure volume levels of every app.

**Comment by meganrogge** [comment]
I believe that was added by @hediet for audio cues.

**Comment by hediet** [comment]
We originally discussed introducing volume settings here: https://github.com/microsoft/vscode/issues/144771
I'm up for removing the setting.

There is also https://github.com/microsoft/vscode/issues/202875, which is related.

**Comment by isidorn** [comment]
I am +1 to remove.
Though I see it is in Stable already as you pointed out. So if we want to be safe we could opt in that setting to telemetry to see if any users actually changed it from the default value.

**Comment by hediet** [comment]
@meganrogge do you mind looking into this? If not, I can ask Alex to put it on the plan for me.

**Comment by meganrogge** [comment]
I will not get to this in June as I'm out for a few weeks and have other stuff. Would be great if you could add it to your plan 🙏🏼.

I'm also not sure how to look at modified settings in telemetry. @isidorn, how can one do that?

**Comment by meganrogge** [comment]
with https://github.com/microsoft/vscode/pull/214163, we can now get telemetry for this

**Comment by meganrogge** [comment]
Looks like people do use this setting, so closing this. 

<img width="625" alt="Image" src="https://github.com/user-attachments/assets/b121b0a3-b33a-4ea4-85e6-d8ec7ab721e2" />
